distribution/docs/src/main/resources/content/_quickstart/quickstart-intro.adoc

```diff
@@ -0,0 +1,25 @@
+:title: Quick Start Tutorial
+:type: quickStart
+:level: intro
+:section: quickStart
+:parent: na
+:status: published
+:summary: Installation of an example instance.
+:order: 00
+
+== {title}
+
+This quick tutorial will enable install, configuring and using a basic instance of ${branding}.
+
+[NOTE]
+====
+This tutorial is intended for setting up a test, demonstration, or trial installation of ${branding}.
+For more complete installation and configuration steps, see <<{managing-prefix}installing,Installing>>.
```

Comment:
Could this be more simply, `For complete installation and configuration steps,...`?

Reply:
fixed